hari/readme.txt

```diff
@@ -0,0 +1,43 @@
+=== Hari ===
+Contributors: Automattic
+Requires at least: 5.8
+Tested up to: 5.8
```

Comment:
this is 5.9, we'll update the plugin so new themes get that by default

Reply:
updated here.  will add it to the list of things to update for the plugin.